### PR TITLE
Plugin refactoring

### PR DESCRIPTION
**Pull request by Yuri2b**
- Extracting images and wrapping then to Gatsby nodes
- Introducing new node types

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  // GraphQL nodes type for entities with type "post"
+  POST_NODE_TYPE: "OnlinesalesPost",
```
Comment (peterliapin) [inline constants.js]: Onlinesales -> OnlineSales - this is a proper way to write the name of the product

File: gatsby-node.js [hunk 2]
```diff
@@ -1,95 +1,129 @@
...
+  IMAGE_FILE_TYPE,
+} = require("./constants");
+
+exports.onPreInit = () =>
+  console.log("Loaded gatsby-source-onlinesales plugin");
+
+const getNodeType = (type) =>
+  type === "post" ? POST_NODE_TYPE : PAGE_NODE_TYPE;
+
+const getNodeLanguage = (language) =>
```
Comment (peterliapin) [inline gatsby-node.js]: @Yuri2b it is OK to keep it this way for now, but moving forward we should consider there could be other languages than ru and en, what happens than?

**Review by peterliapin — CHANGES_REQUESTED** [review]

File: constants.js (hunk 1 above)
Comment (Yuri2b) [reply constants.js]: Fixed

File: gatsby-node.js (hunk 2 above)
Comment (Yuri2b) [reply gatsby-node.js]: I've added simple parsing for 2 chars language code, assuming that CMS provides ISO 639-1 or BCP 47 languge code format (e.g. "en" or "en-US")

**Pull request merged**